oracle-graalvm-17/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM container-registry.oracle.com/graalvm/native-image:17
+
+RUN microdnf --refresh -y install findutils which
```

Comment:
in other images these are used for backwards compatibility, you may want to remove them if they are not needed and adjust the tests and readme table

Reply:
Those packages are certainly not installed by default. Do you mean that they are optional?

Reply:
yes, I don't think maven needs them anymore

Reply:
But `find` is used by `mvn-entrypoint.sh`